build.gradle.kts

```diff
@@ -82,6 +82,7 @@ dependencies {
     compile("si.uom", "si-units", "0.9")
     compile("systems.uom", "systems-common", "0.8")
     compile("com.google.code.gson", "gson", "2.8.5")
+    compile(files("lib/Pathfinder-Java.jar"))
```

Comment:
Is pathfinder not available as a maven dependency? @JacisNonsense 
And does this jar include the native libraries for all four supported platforms?

Reply:
It is on maven, but it isn't built for Linux 32/64, only ARM

Reply:
I added the native build it was missing for Linux. I added a test that generates a path using Pathfinder and it has been running successfully on azure for all platforms.